### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by nobodywasishere**
Changes how the ENV files are required to fix doc generation, and fixes a small deprecation warning.

### CODE REVIEW

File: src/env/darwin.cr
Comment (devnote-dev) [inline src/env/darwin.cr:1]: What improvements does using `skip_file` bring here?

Comment (nobodywasishere) [reply src/env/darwin.cr:1]: It fixes doc generation, small but I figured I'd get it out of the way

Comment (devnote-dev) [reply src/env/darwin.cr:1]: As in `crystal docs` generation? Crimson already documents its usage and components in the CLI.

Comment (nobodywasishere) [reply src/env/darwin.cr:1]: I am aware, I just tend to prefer projects to be able to have API docs working - they're useful beyond using the code as a library (I'm planning on using top-level semantic info for larimar).

**Review by devnote-dev — APPROVED** [review]

**Pull request merged**